.gitignore

```diff
@@ -1,6 +1,244 @@
-# NuGet Packages
-*.nupkg
-
-**/packages/*
-
-!**/packages/build/
+-## Ignore Visual Studio temporary files, build results, and
+ -## files generated by popular Visual Studio add-ons.
+ -
+ -# User-specific files
+ -*.suo
+ -*.user
+ -*.userosscache
+ -*.sln.docstates
+ -
+ -# User-specific files (MonoDevelop/Xamarin Studio)
+ -*.userprefs
+ -
+ -# Build results
+ -[Dd]ebug/
+ -[Dd]ebugPublic/
+ -[Rr]elease/
+ -[Rr]eleases/
+ -[Xx]64/
+ -[Xx]86/
+ -[Bb]uild/
+ -bld/
+ -[Bb]in/
+ -[Oo]bj/
+ -
+ -# Visual Studio 2015 cache/options directory
+ -.vs/
+ -# Uncomment if you have tasks that create the project's static files in wwwroot
+ -#wwwroot/
+ -
+ -# MSTest test Results
+ -[Tt]est[Rr]esult*/
+ -[Bb]uild[Ll]og.*
+ -
+ -# NUNIT
+ -*.VisualState.xml
+ -TestResult.xml
+ -
+ -# Build Results of an ATL Project
+ -[Dd]ebugPS/
+ -[Rr]eleasePS/
+ -dlldata.c
+ -
+ -# DNX
+ -project.lock.json
+ -artifacts/
+ -
+ -*_i.c
+ -*_p.c
+ -*_i.h
+ -*.ilk
+ -*.meta
+ -*.obj
+ -*.pch
+ -*.pdb
+ -*.pgc
+ -*.pgd
+ -*.rsp
+ -*.sbr
+ -*.tlb
+ -*.tli
+ -*.tlh
+ -*.tmp
+ -*.tmp_proj
+ -*.log
+ -*.vspscc
+ -*.vssscc
+ -.builds
+ -*.pidb
+ -*.svclog
+ -*.scc
+ -
+ -# Chutzpah Test files
+ -_Chutzpah*
+ -
+ -# Visual C++ cache files
+ -ipch/
+ -*.aps
+ -*.ncb
+ -*.opendb
+ -*.opensdf
+ -*.sdf
+ -*.cachefile
+ -*.VC.db
+ -
+ -# Visual Studio profiler
+ -*.psess
+ -*.vsp
+ -*.vspx
+ -*.sap
+ -
+ -# TFS 2012 Local Workspace
+ -$tf/
+ -
+ -# Guidance Automation Toolkit
+ -*.gpState
+ -
+ -# ReSharper is a .NET coding add-in
+ -_ReSharper*/
+ -*.[Rr]e[Ss]harper
+ -*.DotSettings.user
+ -
+ -# JustCode is a .NET coding add-in
+ -.JustCode
+ -
+ -# TeamCity is a build add-in
+ -_TeamCity*
+ -
+ -# DotCover is a Code Coverage Tool
+ -*.dotCover
+ -
+ -# NCrunch
+ -_NCrunch_*
+ -.*crunch*.local.xml
+ -nCrunchTemp_*
+ -
+ -# MightyMoose
+ -*.mm.*
+ -AutoTest.Net/
+ -
+ -# Web workbench (sass)
+ -.sass-cache/
+ -
+ -# Installshield output folder
+ -[Ee]xpress/
+ -
+ -# DocProject is a documentation generator add-in
+ -DocProject/buildhelp/
+ -DocProject/Help/*.HxT
+ -DocProject/Help/*.HxC
+ -DocProject/Help/*.hhc
+ -DocProject/Help/*.hhk
+ -DocProject/Help/*.hhp
+ -DocProject/Help/Html2
+ -DocProject/Help/html
+ -
+ -# Click-Once directory
+ -publish/
+ -
+ -# Publish Web Output
+ -*.[Pp]ublish.xml
+ -*.azurePubxml
+ -
+ -# TODO: Un-comment the next line if you do not want to checkin
+ -# your web deploy settings because they may include unencrypted
+ -# passwords
+ -#*.pubxml
+ -*.publishproj
+ -
+  # NuGet Packages
```

Comment:
A bunch of nuget binaries were pushed after I updated the gitignore, but the section of nuget I removed should still be ignored because it's identicle to this.